controllers/user.go

```diff
@@ -322,41 +325,59 @@ func (c *ApiController) UpdateUser() {
 	if id == "" {
 		id = c.GetSessionUsername()
 		if id == "" {
+			logger.Error(goCtx, "missed parameter to update user",
+				"user", user.GetId())
 			c.ResponseUnprocessableEntity(c.T("general:Missing parameter"))
 			return
 		}
 	}
 
 	oldUser, err := object.GetUser(id)
 	if err != nil {
+		logger.Error(goCtx, "failed to get user from database",
+			"id", id,
+			"error", err.Error())
 		c.ResponseInternalServerError(err.Error())
 		return
 	}
 
 	if oldUser == nil {
+		logger.Error(goCtx, "user not found",
+			"id", id)
 		c.ResponseNotFound(fmt.Sprintf(c.T("general:The user: %s doesn't exist"), id))
 		return
 	}
 
 	if oldUser.Owner == "built-in" && oldUser.Name == "admin" && (user.Owner != "built-in" || user.Name != "admin") {
+		logger.Error(goCtx, "tried to update admin user")
 		c.ResponseForbidden(c.T("auth:Unauthorized operation"))
 		return
 	}
 
 	if c.Input().Get("allowEmpty") == "" {
 		if user.DisplayName == "" {
+			logger.Error(goCtx, "tried to set empty display name",
```

Comment:
I belive this message needs more clarity too.
It can become smth like:"UpdateUser failed: empty display name not allowed"

Reply:
done